.travis.yml

```diff
@@ -41,7 +41,7 @@ install:
 
 script:
   - composer phpunit
-  - composer behat
+   - composer behat || composer behat-rerun
```

Comment:
There's an issue with the indentation here.

Reply:
fixed in https://github.com/wp-cli/wp-cli/pull/4906/commits/4a782e0b8822dcd941c7812fb0956cd0bec2545e